GMiguel/User02_test.py

```diff
@@ -1,11 +1,31 @@
-import Project02
-import user02
-import pandas as pd
+import sys
 import unittest
+sys.path.append("C:\\Users\\Stevens User\\Documents\\GitHub\\Team-4-Code\\seeds")
+sys.path.append("C:\\Users\\Stevens User\\Documents\\GitHub\\Team-4-Code\\GMiguel")
+sys.path.append("c:\\Users\\Stevens User\\Documents\\GitHub\\Team-4-Code\\src")
+import user02
```

Comment:
should not use absolute file names rather use relative imports. this will not work on other peoples computers as the paths are not the same. Let me know if you need help with relative imports